### PR TITLE
added mp-b2b availability-notifications

### PR DESCRIPTION
**Pull request by vadym-uskov**
- Copied and refactored tests

### CODE REVIEW

File: tests/api/mp_b2b/glue/availability_endpoints/availability_notifications/positive.robot [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    And Response body parameter should be:    [data][type]    availability-notifications
+    And Response body parameter should be:    [data][id]    ${availability_notification_id}
+    And Response body parameter should be:    [data][attributes][localeName]    ${locale_name_EN}
+    And Response body parameter should be:    [data][attributes][email]    ${yves_user_email}
+    And Response body parameter should be:    [data][attributes][sku]    ${concrete_product_with_alternative_sku}
+    And Response body has correct self link for created entity:    ${availability_notification_id}
+    [Teardown]    Run Keywords    I send a DELETE request:    /availability-notifications/${availability_notification_id}
+    ...  AND    Response status code should be:    204
+
+Subscribe_to_availability_notifications_with_non_existing_email
```
Comment (OKrivtsova) [inline tests/api/mp_b2b/glue/availability_endpoints/availability_notifications/positive.robot:54]: test fails on staging
image

Comment (vadym-uskov) [reply tests/api/mp_b2b/glue/availability_endpoints/availability_notifications/positive.robot:54]: - added random value for email
- created the bug-ticket: https://spryker.atlassian.net/browse/CC-16611

File: tests/api/b2b/glue/availability_endpoints/availability_notifications/positive.robot [hunk 2]
```diff
@@ -51,14 +51,14 @@ Subscribe_to_availability_notifications_for_customer
...
-    When I send a POST request:    /availability-notifications    {"data": {"type": "availability-notifications","attributes": {"sku": "${concrete_product_with_alternative_sku}","email": "test1234546@gmail.com"}}}
+    When I send a POST request:    /availability-notifications    {"data": {"type": "availability-notifications","attributes": {"sku": "${concrete_product_with_alternative_sku}","email": "test-${random}@gmail.com"}}}
     And Save value to a variable:    [data][id]    availability_notification_id
     Then Response status code should be:    201
     And Response reason should be:    Created
     And Response body parameter should be:    [data][type]    availability-notifications
     And Response body parameter should be:    [data][id]    ${availability_notification_id}
     And Response body parameter should not be EMPTY:    [data][attributes][localeName]
-    And Response body parameter should be:    [data][attributes][email]    test1234546@gmail.com
+    And Response body parameter should be:    [data][attributes][email]    test-${random}@gmail.com
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/availability_endpoints/availability_notifications/positive.robot:61]: please use + instead of -

Comment (vadym-uskov) [reply tests/api/b2b/glue/availability_endpoints/availability_notifications/positive.robot:61]: fixed

File: tests/api/mp_b2b/glue/availability_endpoints/availability_notifications/positive.robot (hunk 1 above)
Comment (vadym-uskov) [reply tests/api/mp_b2b/glue/availability_endpoints/availability_notifications/positive.robot:54]: - test fixed
- bug was deleted

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**